src/poetry/config/config.py

```diff
@@ -135,6 +135,7 @@ class Config:
         "keyring": {
             "enabled": True,
         },
+        "system-truststore": True,
```

Comment:
Default to `False`.

Reply:
I can agree, but if we go with `False`, we should set a date for when it goes `True` by default.

Reply:
Switched and added a TODO with a flip date proposal.